doc/changes.md

```diff
@@ -1,5 +1,124 @@
 # Changelog
 
+## Release v0.11.0 (2024/02/01)
+
+[![Digital Object Identifier for PyGMT v0.11.0](https://zenodo.org/badge/DOI/10.5281/zenodo.10578540.svg)](https://doi.org/10.5281/zenodo.10578540)
+
+### Highlights
+
+* Intro for table inputs in PyGMT
+* Gallery example for Chorophleth map
+* Easy access to remote datasets of planetary reliefs (Moon, Venus, Mars, Mercury, Pluto)
+* `Figure.text`: Allow lists as input type for the `angle`, `font`, and `justify` parameters
```

Comment:
I agree with @yvonnefroehlich that 2-4 highlights are a good number and have applied the suggestions by @yvonnefroehlich and @weiji14.

I also would like to highlight the changes in PR https://github.com/GenericMappingTools/pygmt/pull/2930, which speed up each module call by about 0.1 seconds. The performance improvement seems small but it can significantly speed up PyGMT scripts that are not computationally intensive. For example, changes in PR #2930 have reduced our CI testing times from 250 seconds to 85 seconds (https://github.com/GenericMappingTools/pygmt/pull/2930#issue-2058474856).

If this change is highlighted, then maybe remove the highlights about Chorophleth and `Figure.text`?

```suggestion
* Intro for table inputs in PyGMT
* Easy access to remote datasets of planetary reliefs (Moon, Venus, Mars, Mercury, Pluto)
* Faster PyGMT by ~0.1 seconds for each module call
```

Reply:
We can combine the tutorial (table inputs) and gallery example (choropleth) in one line perhaps, and remove the `Figure.text` one

Reply:
I suggested the `Figure.text` improvement, because I was hoping to complete PR #2760. But as this will be unfortunately not the case, I feel we can remove this from the highlights section and move it to the nice enhancements section in the release announcement.

Reply:
Regarding the choropleth map: So far, I know, it is not so straight forward to create such a visualization directly in GMT. Also, we show here the integration with geopandas. Thus, I feel it would be nice to mention it.